pkg/reconciler/common/releases.go

```diff
@@ -62,9 +62,22 @@ func TargetVersion(instance v1alpha1.KComponent) string {
 	return version
 }
 
-// TargetManifest returns the manifest for the TargetVersion
+// TargetManifest returns the default manifest for the TargetVersion or the manifest for the TargetVersion specified
+// with spec.manifests
 func TargetManifest(instance v1alpha1.KComponent) (mf.Manifest, error) {
-	return getManifestWithVersionValidation(TargetVersion(instance), instance)
+	version := TargetVersion(instance)
+	manifestsPath := targetManifestPath(version, instance)
+	return getManifestWithVersionValidation(version, manifestsPath, instance)
+}
+
+// TargetAdditionalManifest returns the manifest for the TargetVersion specified with spec.additionalManifests.
+func TargetAdditionalManifest(instance v1alpha1.KComponent) (mf.Manifest, error) {
+	version := TargetVersion(instance)
+	additionalManifestsPath := additionalManifestPath(version, instance)
+	if additionalManifestsPath == "" {
+		return mf.Manifest{}, nil
+	}
+	return getManifestWithVersionValidation(version, additionalManifestsPath, instance)
```

Comment:
`additionalManifestPath()` could return comma separated manifests. If `additionalManifestsPath` contains comma, `getManifestWithVersionValidation()` could work properly?

Reply:
Yes, if there are multiple paths, `getManifestWithVersionValidation` works fine with parsing and reading them, as long as they are concatenated with comma.